test/Tester/PSClientTests/PSClientTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Orleans;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using OrleansPSUtils;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using UnitTests.Tester;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Tester
+{
+    public class PowershellHostFixture : DefaultClusterFixture
```

Comment:
you probably don't want to inherit from DefaultTestCluster fixture, and instead use it's base class and start the Silo cluster but not necessarily the grain client (I believe there's an overload to TestCluster's constructor where you just use the ClusterConfiguration but just pass a null ClientConfiguration). You can take the ClusterConfiguration object from the TestClusterOptions.ClusterConfiguration property.